### DISCUSSION TITLE
GitHub opens far too many tabs

### DISCUSSION THREAD

**Discussion by uaru** [community/community#38747] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The fact, that today my card was declined is not really a good reason to open 500+ tabs in my browser, one every couple of minutes, with the message that the card was declined and I should update it. Ok, I got it, I will do it after the work. 

While I was writing this, two tabs were opened by Github! 🤦‍♂️

**Comment by ossanna16** [comment]
Hi there @uaru and welcome to our community! Thank you for the feedback!

**Comment by uaru** [comment]
<img width="597" alt="Screen Shot 2022-12-15 at 14 08 18" src="https://user-images.githubusercontent.com/2102032/207777775-38358879-1883-43b8-aba4-d3bc444d7721.png">

All those tabs are the same Codespace...
